projects/components/form/src/form.component.ts

```diff
@@ -390,4 +407,32 @@ export class PsFormComponent implements OnChanges, OnInit, OnDestroy {
     }
     this._savebar.cd.markForCheck();
   }
+
+  private setupIntersectionObserver(ex: IPsFormException) {
+    if (ex) {
+      if ('IntersectionObserver' in window) {

```

Comment:
All browsers we support also support IntersectionObserver. Only IE doesn't, but our components don't fully work with IE anyway.